SPECS/php/php.spec

```diff
@@ -32,7 +32,7 @@
 %global with_qdbm     0
 Summary:        PHP scripting language for creating dynamic web sites
 Name:           php
-Version:        8.3.6
+Version:        8.3.8
 Release:        1%{?dist}
 # All files licensed under PHP version 3.01, except
```

Comment:
README.REDIST.BINS file in root of 8.3 has an additional entry, not in 8.1: 
18. avifinfo (ext/standard/libavifinfo) see ext/standard/libavifinfo/LICENSE 
I think we build and redistribute libavifinfo as part of php, at least in php-devel (The php-devel package contains the files needed for building PHP extensions).  And libavifinfo is under ext/standard, which seems to be php extensions.  So should we add %files and %license sections for avif?